### PR TITLE
Update pyhack-camp-manual_en.md

### PR DESCRIPTION
**Pull request by ojii**

### CODE REVIEW

File: pyhack-camp-manual_en.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Overview
 
 - Instructions for Pyhack camp we hold twice in a year
-- Assumption: to have the pension all to ourselves
+- We have pension all to ourselves
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md]:
```suggestion
- We will have pension all to ourselves
```
I think it's not guaranteed that we have the pension all to ourselve as stated in the original Japanese text '貸し切りでの開催を前提にしている'.

File: pyhack-camp-manual_en.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 # Venue & Equipments
 
 ## Internet
 
-- The pension uses B Flet's Hikari.
-- WiFi (Password to be notified at the venue)
+- The pension has B Flet's Hikari internet.
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:10]: Accepted.

File: pyhack-camp-manual_en.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+- WiFi is available (Password to be notified at the venue)
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md]:
```suggestion
- WiFi is available (Password to be notified at the venue).
```

File: pyhack-camp-manual_en.md [hunk 4]
```diff
@@ -17,8 +17,8 @@
     - Until 30 minutes before breakfast and dinner
     - Do not leave your PC nor books when you don't use the room.
 - Some power cables available.
-- You can connet your PC to a large TV in the room.
-    - 1 VGA cable is equipped.
+- You can connect your PC to a large TV in the room.
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:20]: Accepted.

File: pyhack-camp-manual_en.md [hunk 5]
```diff
@@ -17,8 +17,8 @@
...
+    - A VGA cable is available.
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:21]: Accepted.

File: pyhack-camp-manual_en.md [hunk 6]
```diff
@@ -60,14 +60,14 @@
 ## Day 1
 
 - ca. 15:00, Arrival & instruction
-- 19:00, Dinner (TBC)
+- ~19:00, Dinner
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:63]:
```suggestion
- 19:00, Dinner (Time could be changed.)
```
My initial translation 'TBC' to '時間変更の可能性あり' was too rough.

File: pyhack-camp-manual_en.md [hunk 7]
```diff
@@ -60,14 +60,14 @@
...
 - ca. 19:30, Self-introduction
 
 ## Day 2 & Middle Day
 
 - 8:00, Breakfast
 - ca. 12:30, Lunch (optional, with extra fee)
-- 19:00, Dinner (TBC)
+- ~19:00, Dinner
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:70]:
```suggestion
- 19:00, Dinner (Time could be changed.)
```
My initial translation 'TBC' to '時間変更の可能性あり' was too rough.

File: pyhack-camp-manual_en.md [hunk 8]
```diff
@@ -94,9 +94,9 @@
 
 # Arrival by Car
 
-- 6 cars can be parked at the pension. You may come by your car and motorbike.
+- 6 cars can be parked at the pension. You may come by your car or motorbike.
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:97]: Accepted.

File: pyhack-camp-manual_en.md [hunk 9]
```diff
@@ -94,9 +94,9 @@
...
 - Leave your car key on the counter in the dining room when you park your car in the parking area.
-- Pension staffs would move your car if necessary.
+- Pension staffs will move your car if necessary.
```
Comment (ytknzw) [inline pyhack-camp-manual_en.md:99]: Accepted.

**Review by ytknzw — COMMENTED** [review]
Thank you for your updates, @ojii ! Please take a look at my comments.

File: pyhack-camp-manual_en.md (hunk 1 above)
Comment (ojii) [reply pyhack-camp-manual_en.md]: updated

File: pyhack-camp-manual_en.md (hunk 6 above)
Comment (ojii) [reply pyhack-camp-manual_en.md:63]: That's why I added the `~` meaning "around/circa/roughly" (most non Japanese also won't expect the times to be strictly kept www)

**Pull request merged**